platform/security/security-services-app/src/main/resources/features.xml

```diff
@@ -127,6 +127,7 @@
 
     <feature name="security-sts-ldaplogin" install="manual" version="${project.version}"
              description="Security STS JAAS LDAP Login.">
+        <bundle>mvn:org.apache.commons/commons-pool2/2.5.0</bundle>
```

Comment:
❓ why is this needed?  Isn't commons-pool2 a simple jar not a bundle?

Reply:
Its a bundle and a new dependency we don't directly use any version of commons-pool elsewhere in DDF. We use some libraries that do use the 1.6 commons-pool.